src/parts/_forms.scss

```diff
@@ -38,6 +38,11 @@ textarea {
   resize: vertical;
 }
 
+select {
+  background: url("data:image/svg+xml;utf8,<svg fill='#{$form-text}' viewBox='0 0 140 140' width='24' height='24' xmlns='http://www.w3.org/2000/svg'><g><path d='m121.3,34.6c-1.6-1.6-4.2-1.6-5.8,0l-51,51.1-51.1-51.1c-1.6-1.6-4.2-1.6-5.8,0-1.6,1.6-1.6,4.2 0,5.8l53.9,53.9c0.8,0.8 1.8,1.2 2.9,1.2 1,0 2.1-0.4 2.9-1.2l53.9-53.9c1.7-1.6 1.7-4.2 0.1-5.8z'/></g></svg>") calc(100% - 12px) 50% / 12px no-repeat $background-alt;
```

Comment:
I don't think this line isn't coming through correctly. Here's the compiled CSS I see on the preview site:
```
select {
    background: url("data:image/svg+xml;utf8,<svg fill='#ffffff' viewBox='0 0 140 140' width='24' height='24' xmlns='http://www.w3.org/2000/svg'><g><path d='m121.3,34.6c-1.6-1.6-4.2-1.6-5.8,0l-51,51.1-51.1-51.1c-1.6-1.6-4.2-1.6-5.8,0-1.6,1.6-1.6,4.2 0,5.8l53.9,53.9c0.8,0.8 1.8,1.2 2.9,1.2 1,0 2.1-0.4 2.9-1.2l53.9-53.9c1.7-1.6 1.7-4.2 0.1-5.8z'/></g></svg>") calc(100% - 12px) 50%/12px no-repeat #161f27;
    padding-right: 35px
}
```

I don't end up seeing the arrow:
![image](https://user-images.githubusercontent.com/6742479/56462479-8760bc80-6378-11e9-8df7-b216fd98b2aa.png)

You can see the preview site here:
https://deploy-preview-58--watercss.netlify.com/

Does it seem like something is wrong with the compiled CSS I've listed above? If you can tell me what's wrong, I can give you a hand to get this resolved!

Reply:
@kylejrp The problem is the fill property of the `svg`.
If I use `<svg fill='white'` is works. I try to figure out how to pass in a SASS-variable as fill property.

(The right icons was shown because of my browser cache)

Reply:
@IL4Miy I'm pretty sure CSS has a `fill` property. Maybe that'll work?

https://css-tricks.com/almanac/properties/f/fill/

Reply:
@kognise [This](http://fraudoudou.at/article/color-variables-svg-sass/) article describes a way to make this work for me in Chrome and Safari by using 
```
$form-text-rgb: "rgb(" +
   red($form-text) + ", " +
   green($form-text) + ", " +
   blue($form-text) +
")";
```
And then adding `fill='#{$form-text-rgb}'` to the `select` tag.
Unfortunately this will not work in IE and we need a ruby function to make a URL encoding.
I think this is a bit too overkill. Maybe I find another solution.

Reply:
@kognise @kylejrp Should be fixed now. It works for me in Chrome and Safari. 
Sadly I can't test it on IE. Does is work on your machine?

Reply:
Looks great on my browser:
![image](https://user-images.githubusercontent.com/6742479/56463364-82573980-6387-11e9-8e65-b91593be1065.png)

I opened IE11 for the first time and the demo page has a few things wrong with it - I don't think we have to worry about compatibility in this issue. I'll open up a separate issue.

Reply:
Looks great! Question: why do we have to use `$form-text-rgb` instead of just `$form-text`? Maybe I'm being stupid

Reply:
I am not using this in the current solution. The problem is the `#` because I want to use a SASS variable in the `svg` string (`#{var-name}`). By converting HEX to RGB this however fixes the problem. 

In the current solution the `#` is simply encoded.
However it looks a bit weird in Firefox. Maybe I can adjust this.
I also have an idea for the IE11 bug but it has to wait till tomorrow :)

Reply:
Shouldn't you be able to do `#{$form-text}`

Reply:
@kognise No, this was the problem in the first commit where you don't see the arrow :)

I think it is okay in Firefox. With the dark theme the option list is dark and with the light theme it is light. Maybe you can check this in Firefox and give me a feedback if you like it.

---
#### IE11 Bug
```
select::-ms-expand {
    display: none;
}
```
This should fix the issue in Internet Explorer 11 but I am not able to test it. Maybe you can test this?

Reply:
I'm on Linux but there's this amazing service [Browserling](https://www.browserling.com/) that lets you test websites in any browser. I can try using it

Reply:
Cool, I did not know something like that exist.
I try it out this evening.

Maybe this can also be used to create a list of all supported browsers?

Reply:
@kognise 
I figured it out so you don't have to find a solution for the IE11 fix. It is working in IE11 now.
The problem was the encoding of the svg. If it is not encoded, IE11 can't handle it.

I don't want to use an external ruby function, so I found [this](https://gist.github.com/JacobDB/0ffffaf8e772c12acf7102edb8a302be) SASS function.
With this one it is working.

I will commit the changes in the next days.

Reply:
Why should we use a SASS function for this instead of just embedding it encoded? It's already totally unreadable

Reply:
Because we pass `form-text` as variable to the `fill` property.
Maybe there is another solution but I can't figured it out yet.

Reply:
Okay, so there's two problems:
1. We can't easily change the stroke color of a backround SVG with SASS
2. An inline SVG is difficult to understand and read

I think we should focus on merging this change in for now and making it work nicer later. My proposition:
1. Use two hardcoded SVGs, one that's black and one that's white. We could use a variable like `$down-arrow-path` that points to the hardcoded SVG. We can open a separate issue to style these with our SASS variables.
2. Reference the SVGs as an external file

Then, our final code becomes:
```
select {
    background: url($down-arrow-path);
    padding-right: 35px;
}
```

Later, we can modify our `gulpfile.js` to use [gulp-sass-inline-svg](https://www.npmjs.com/package/gulp-sass-inline-svg) to inline the SVGs at build time.

Reply:
This is a very good idea :)

Reply:
@IL4Miy Would you be able to modify your pull request to include:
- Create two SVG files, one for the light theme and one for the dark theme
- Change the `background:` rule to reference the `$down-arrow-path` variable
- Fill in the `$down-arrow-path` variable in `light.scss`/`dark.scss`

We should be able to merge this after those changes! Thank you so much for your contribution. 

(Alternatively, there's no pressure for you to make these changes. If you would like someone else to finish up this work, please let us know!)

Reply:
@kylejrp Yes I can do this but I can't modify the `gulpfile` as you mentioned above because I am not familiar with this.

Reply:
@IL4Miy Thanks! We'll take care of inlining the SVGs with Gulp in a separate issue after we merge this change.